presto-tests/src/main/java/com/facebook/presto/tests/QueryTemplate.java

```diff
@@ -79,11 +81,12 @@ private void checkQueryHasAllParametersReplaced(String query)
         }
     }
 
-    public List<String> replaceAll(List<Parameter>... parametersLists)
+    @SafeVarargs
+    public final Stream<String> replaceAll(List<Parameter>... parametersLists)
```

Comment:
We mainly use the `forEach` method on the result of this method in usages from before this PR, so it doesn't make a difference for the previous usages (Stream has `forEach`). This PR needs the Stream, not the list.

Having a Stream and converting to a List on-demand is less computation than having a List and then `.stream()`-ing. 

Also, note that we don't need a List in any of the usages so far. If a List is needed, the callers can `.collect(toList())`.